samples/js/node/hello_classification/README.md

```diff
@@ -1,6 +1,6 @@
 # Hello Classification NodeJS Sample
 
 Run:
-`node hello_classification.js ../../assets/models/v3-small_224_1.0_float.xml ../../assets/images/coco224x224.jpg AUTO`
+`node hello_classification.js ../../assets/models/classification.xml ../../assets/images/coco.jpg AUTO`
```

Comment:
Where can I find this classification.xml model?

Reply:
README.md extended